codecov.yml

```diff
@@ -0,0 +1,6 @@
+coverage:
```

Comment:
Ah, I am not sure if integrating codecov with Gitter is necessary. What are your views?

Reply:
Okay, I think Codecov is not much necessary than the CI tests webhooks.

Reply:
Done